### ISSUE TITLE
num_epoch

### ISSUE THREAD

**Issue by NoahBU** [toandaominh1997/EfficientDet.Pytorch#51]
hi, thanks for sharing!
In train.py, paramter "num_epoch" defaults 500. I thought the total iteration num is 500*1000, but it seem not. Could you explain this param ?

**Comment by toandaominh1997** [comment]
@NoahBU Because I have saved weights for each epoch, I put a lot of epoch so my server can work during my free time. I can choose best weights based on the loss and accuracy of the validation set in each epoch:v
